cmd/restic/cmd_backup.go

```diff
@@ -278,16 +263,7 @@ func (cmd CmdBackup) Execute(args []string) error {
 		return nil
 	}
 
-	err = arch.Cache().RefreshSnapshots(s, newCacheRefreshProgress())
-	if err != nil {
-		return err
-	}
-
-	fmt.Printf("loading blobs\n")
-	err = arch.Preload()
-	if err != nil {
-		return err
-	}
+	// TODO: load index
```

Comment:
what's this? this sounds like important functionality :-) is this ok to be missing?